### PR TITLE
Final Documentation Changes

### PR DESCRIPTION
**Pull request by TAsif**
# Documentation Changes

### CODE REVIEW

File: docs/Reflection/Reflection.tex [hunk 1]
```diff
@@ -14,39 +14,126 @@
...
+
+\noindent The final design and implementation contain two pieces: the front-end, which is the client, and the back-end, which is the server. As the client is user-facing and directly interacts with it, the design is swayed by feedback and how the application "feels" to use. Whereas the server is not directly interacted with by the users, they only see the results of it. Given this, the client was changed more than the server.
+
+\subsection{Client Design and Implementation}
+Going deeper into the client's design, we, as developers, are also users of the application, where we had a decent idea of how the client should look and the overall flow of it, which is what we started with. Using Figma, 7eam could display their thoughts as pictures, where each component should be, and what pages are needed. Going from Figma to the actual initial implementation had a few colour changes and made the application feel smooth. However, after publishing the application and collecting feedback, more changes had to be made:
+\begin{itemize}
+	\item Additional preferences were added
+	\item History functionality was added
+	\item Layout was modified on some pages
+	\item Adding friends is clunky
```
Comment (JarrodColwell) [inline docs/Reflection/Reflection.tex]: Maybe reword as "Improve friends feature"

File: docs/Reflection/Reflection.tex [hunk 2]
```diff
@@ -14,39 +14,126 @@
...
+\end{itemize}
+Making the difference between us as developers and users abundantly clear. We were extremely comfortable as we had been looking at the pages endlessly, but a user coming in for the first time had to learn the layout. 7eam wanted to make that learning process seamless and to use the application faster. Implementing and designing the client is a constant reiterative process. Users can provide more detailed feedback on what they expect as they use the application more. The client's current iteration cannot be considered the final and is expected to evolve continually. Still, the abovementioned changes are how the client changed from the first iteration to the current one.
```
Comment (JarrodColwell) [inline docs/Reflection/Reflection.tex]: 7eam wanted to make that learning process seamless and to use the application faster. Replace with "7eam wanted to make sure that the users were able to learn how to use the application in a fast and simple way."

Comment (JarrodColwell) [reply docs/Reflection/Reflection.tex]: "current iteration cannot be considered **~~the~~** final"

File: docs/UserGuide/UserGuide.tex [hunk 3]
```diff
@@ -35,5 +29,10 @@
 
 \maketitle
 
+\section{How to Use}
+\progname ~is an easy to use web application, you first sign-up with either an email and password, or through Google authentication. After logging into the website for the first time you should click "Preferences" and set those to let us know which flicks to recommend. You can also add friends by going to "Social" and then creating groups and inviting friends to join them. After going into a group the owner can start a voting session, where our algorithm will provide flicks based off of every user's preferences' in the group. However, just because one person disliked a genre does not mean that genre will not show up, it depends on everyone else's preferences.
```
Comment (madhi-naga) [inline docs/UserGuide/UserGuide.tex]: Fix quotes syntax for Social and Preferences. Also is "~" supposed to be present?

File: docs/Reflection/Reflection.tex [hunk 4]
```diff
@@ -14,39 +14,100 @@
...
+
+\section{Changes in Response to Feedback}
+
+\plt{Summarize the changes made over the course of the project in response to
+feedback from TAs, the instructor, teammates, other teams, the project
+supervisor (if present), and from user testers.}
+
+\plt{For those teams with an external supervisor, please highlight how the feedback 
+from the supervisor shaped your project.  In particular, you should highlight the 
+supervisor's response to your Rev 0 demonstration to them.}
```
Comment (TAsif) [inline docs/Reflection/Reflection.tex:49]: Remove this after the section is filled out

File: docs/Reflection/Reflection.tex [hunk 5]
```diff
@@ -14,39 +14,100 @@
...
+
+\subsection{SRS and Hazard Analysis}
+
+\subsection{Design and Design Documentation}
+
+\subsection{VnV Plan and Report}
+
+\section{Design Iteration (LO11)}
+
+\noindent The final design and implementation contain two pieces: the front-end, which is the client, and the back-end, which is the server. As the client is user-facing and directly interacts with it, the design is swayed by feedback and how the application "feels" to use. Whereas the server is not directly interacted with by the users, they only see the results of it. Given this, the client was changed more than the server.
```
Comment (TAsif) [inline docs/Reflection/Reflection.tex:59]:
```suggestion
\noindent The final design and implementation contain two pieces: the front-end, which is the client, and the back-end, which is the server. As the client is user-facing and directly interacts with it, the design is swayed by feedback and how the application ``feels" to use. Whereas the server is not directly interacted with by the users, they only see the results of it. Given this, the client was changed more than the server.
```

**Review by madhi-naga — DISMISSED** [review]

**Review by JarrodColwell — APPROVED** [review]

**Review by madhi-naga — APPROVED** [review]
